examples/db-templates/mongodb-template.json

```diff
@@ -1,127 +1,150 @@
 {
-  "apiVersion": "v1beta1",
   "kind": "Template",
+  "apiVersion": "v1beta3",
   "metadata": {
+    "name": "mongodb",
+    "creationTimestamp": null,
```

Comment:
@soltysh this is done by the conversion tool we use, I don't want to fine tune these as you will have to do it every time you run that tool. We should rather focus on fixing the tool, which we will do post-3.0.